examples/graphviz_drawing/plot_grid.py

```diff
@@ -0,0 +1,24 @@
+"""
+2D Grid
+=======
+
+Illustrate nx_agraph.write_dot in conjunction with the Graphviz command line
```

Comment:
Turn this into a link to the docs.

Reply:
Reference to `write_dot` should now be a valid link